object/object.go

```diff
@@ -35,10 +46,62 @@ func (v *Version) IsLater(other *Version) bool {
 		return true
 	}
 
-	// Either v.Version < other.Version or other.Pid > v.Pid
+	// Either v.Version < other.Version, other.Pid > v.Pid, or the versions are equal.
 	return false
 }
 
+// Equal returns true if and only if the Version scalars and PIDs are equal. Nil is
+// considered to be the zero valued Version.
+func (v *Version) Equal(other *Version) bool {
+	// If other is nil, then we assume it represents the zero-valued version.
+	if other == nil {
+		other = &VersionZero
+	}
+	return v.Version == other.Version && v.Pid == other.Pid
+}
+
+// Concurrent returns true if and only if the Version scalars are equal but the PIDs are
+// not equal. Nil is considered to be the zero valued Version.
+func (v *Version) Concurrent(other *Version) bool {
+	// If other is nil, then we assume it represents the zero-valued version.
+	if other == nil {
+		other = &VersionZero
+	}
+	return v.Version == other.Version && v.Pid != other.Pid
+}
```

Comment:
Also the same as "conflict", though this isn't the only kind of conflict, e.g. branches